CHANGELOG.txt

```diff
@@ -1,3 +1,7 @@
+# 2.1.0 - 2016-04-08
```

Comment:
In the future, add `unreleased` instead, it's up to the maintainer to determine version numbers and all that. Still, this is helpful, so :+1: thanks!